magicctapipe/io/tests/test_io.py

```diff
@@ -19,619 +21,721 @@
+                    "migration_bins",
+                    "source_offset_bins",
+                    "bkg_fov_offset_bins",
+                    "file_names",
+                ]
+            )
         )
-    )
-    assert len(irf["effective_area"][0][0]) > 0
-    assert "psf_table" not in list(irf.keys())
-    assert "background" not in list(irf.keys())
-    assert set(list(header.keys())).issubset(
-        set(
-            [
-                "TELESCOP",
-                "INSTRUME",
-                "FOVALIGN",
-                "QUAL_CUT",
-                "EVT_TYPE",
-                "DL2_WEIG",
-                "IRF_OBST",
-                "GH_CUT",
-                "GH_EFF",
-                "GH_MIN",
-                "GH_MAX",
-                "RAD_MAX",
-                "TH_EFF",
-                "TH_MIN",
-                "TH_MAX",
-            ]
+        assert len(irf["effective_area"][0][0]) > 0
+        assert "psf_table" not in list(irf.keys())
+        assert "background" not in list(irf.keys())
+        assert set(list(header.keys())).issubset(
+            set(
+                [
+                    "TELESCOP",
+                    "INSTRUME",
+                    "FOVALIGN",
+                    "QUAL_CUT",
+                    "EVT_TYPE",
+                    "DL2_WEIG",
+                    "IRF_OBST",
+                    "GH_CUT",
+                    "GH_EFF",
+                    "GH_MIN",
+                    "GH_MAX",
+                    "RAD_MAX",
+                    "TH_EFF",
+                    "TH_MIN",
+                    "TH_MAX",
+                ]
+            )
         )
-    )
-    assert header["DL2_WEIG"] == "simple"
-    assert header["EVT_TYPE"] == "software"
+        assert header["DL2_WEIG"] == "simple"
+        assert header["EVT_TYPE"] == "software"
 
+    def test_load_irf_files_exc(self, temp_irf_exc):
+        """
+        Check on exception (FileNotFound)
+        """
+        with pytest.raises(
+            FileNotFoundError,
+            match="Could not find any IRF data files in the input directory.",
+        ):
+            _, _ = load_irf_files(str(temp_irf_exc))
 
-def test_load_irf_files_exc(temp_irf_exc):
-    """
-    Check on exception (FileNotFound)
-    """
-    with pytest.raises(
-        FileNotFoundError,
-        match="Could not find any IRF data files in the input directory.",
-    ):
-        _, _ = load_irf_files(str(temp_irf_exc))
 
+class TestDL1LST:
+    @pytest.mark.dependency()
+    def test_load_lst_dl1_data_file(self, dl1_lst):
+        """
+        Check on LST DL1
+        """
+        for file in dl1_lst:
+            events, _ = load_lst_dl1_data_file(str(file))
+            assert "event_type" in events.columns
+            assert "slope" in events.columns
+            assert "az_tel" not in events.columns
+            events = events.reset_index()
+            s = events.duplicated(subset=["obs_id_lst", "event_id_lst"])
+            s1 = ~s
```

Comment:
I think ```assert np.all (s==False)``` would work as well instead of those two lines, but fine to keep it as it is if you prefer the current version

Reply:
done

Reply:
sorry, this actually triggered linter because I put unnecessary space between the name of the function and the bracket, it should be:
```assert np.all(s==False)```

Reply:
No worries, I will run pre-commit after a couple of major fixes and it will get fixed :)